### PR TITLE
[ACS-7690] Saving adding o a new hold

### PR DESCRIPTION
**Pull request by DaryaBalvanovich**
**Please check if the PR fulfills these requirements**

https://hyland.atlassian.net/browse/ACS-7690

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
no endpoint for create holds

**What is the new behaviour?**
integrate create and assign endpoints

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export class LegalHoldService {
     constructor(private readonly apiService: AlfrescoApiService) {}
 
     /**
-     * Gets the list of holds.
+     * Gets the list of holds to the node.
```
Comment (MichalKinas) [inline lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts:37]:
```suggestion
     * Gets the list of holds available in the file plan.
```

File: lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts [hunk 2]
```diff
@@ -87,4 +87,26 @@ export class LegalHoldService {
...
+    }
+
+    /**
+     * Create list of holds.
+     *
+     * @param filePlanId The identifier of a file plan. You can also use the -filePlan- alias.
+     * @param holds Holds to create
+     * @returns List of created holds Observable<HoldPaging[]>
+     */
+    createHolds(filePlanId: string, holds: Hold[]): Observable<HoldPaging> {
```
Comment (MichalKinas) [inline lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts:109]: There is no point in having two separate methods you can instead modify the return type to something like `Observable<HoldPaging | HoldEntry>` there should be some examples already existing in the repo

File: lib/js-api/src/api/gs-core-rest-api/api/legal-hold.api.ts [hunk 3]
```diff
@@ -113,4 +112,50 @@ export class LegalHoldApi extends BaseApi {
...
+    }
+
+    /**
+     * Create list of new holds
+     *
+     * @param filePlanId The identifier of a file plan. You can also use the -filePlan- alias.
+     * @param holds Array of holds
+     * @returns Promise<HoldPaging>
+     */
+    createHolds(filePlanId = '-filePlan-', holds: Hold[]): Promise<HoldPaging> {
```
Comment (MichalKinas) [inline lib/js-api/src/api/gs-core-rest-api/api/legal-hold.api.ts:146]: Same as for the service

File: lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts [hunk 4]
```diff
@@ -87,4 +87,26 @@ export class LegalHoldService {
...
     }
+
+    /**
+     * Create hold.
+     *
+     * @param filePlanId The identifier of a file plan. You can also use the -filePlan- alias.
+     * @param holds Holds to create
+     * @returns List of created holds Observable<HoldEntry[]>
+     */
+    createHold(filePlanId: string, holds: Hold[]): Observable<HoldEntry> {
```
Comment (MichalKinas) [inline lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts:98]: Also what about the docs for the service?

**Review by MichalKinas — CHANGES_REQUESTED** [review]

File: lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts (hunk 1 above)
Comment (DaryaBalvanovich) [reply lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts]: done

File: lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts (hunk 2 above)
Comment (DaryaBalvanovich) [reply lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts:109]: As we alined with you and @DenysVuika should be 2 separate methods

File: lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts (hunk 4 above)
Comment (DaryaBalvanovich) [reply lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts:98]: Added

**Pull request merged**